vendor/session-manager-plugin/Makefile

```diff
@@ -0,0 +1,32 @@
+SHELL := /bin/bash
+
+include ../../tasks/Makefile.vendor_includes
+
+ifeq ($(OS), darwin)
+	FILENAME_PATH=mac/sessionmanager-bundle.zip
+else
+	FILENAME_PATH=linux_64bit/session-manager-plugin.rpm
+endif
+
+export APK_BUILD_TEMPLATE ?= APKBUILD.github-binary
+
+export PACKAGE_TYPES_DISABLED = rpm
```

Comment:
I added this since I don't think we're installing any other rpms anywhere in this repo. I saw it ignored for a few other packages.